atqa-framework/browsers.json

```diff
@@ -8,7 +8,7 @@
         "path": "/",
         "screenResolution": "1920x1080x24",
         "volumes": [
-          "/home/runner/work/waves-keeper/waves-keeper/atqa-framework/extension:/home/selenium"
+          "/Users/a1/waves/waves-keeper/atqa-framework/extension:/home/selenium"
```

Comment:
As we discussed, this file should be renamed `browser.example.json`.
And should be copied to `browser.json` during deployment.